extension-control/extensions.json

```diff
@@ -62,6 +62,13 @@
         }
     ],
     "deprecated": {
+        "AnuragKochar.that-typescript-extension" : {
+            "disallowInstall": true,
+            "extension": {
+                "id": "AnuragKochar.that-typescript-extension",
```

Comment:
Hi there @Anurag-Kochar-1 👋, thanks for your PR. The field under `extension.id` should be used as the replacement (if applicable), so please either remove it or change to your preferred replacement.

I also notice the extension is not on https://open-vsx.org anymore, so please note that this will only affect existing users who should see an in-editor nudge towards migrating or uninstalling.